testing/ios/IosUnitTests/Tests/FlutterEngineConfig.xcconfig

```diff
@@ -1,3 +1,3 @@
 FLUTTER_ENGINE[arch=x86_64]=ios_debug_sim_unopt
 FLUTTER_ENGINE[arch=arm64]=ios_debug_sim_unopt_arm64
-FLUTTER_ENGINE=ios_debug_sim_unopt
+FLUTTER_ENGINE=ios_debug_sim_unopt_arm64
```

Comment:
There are more folks running these tests on ARM Macs now than x64.

Reply:
does the previous 2 line do anything? is it supposed to pick the right engine based on which mac you are running on?